.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
   "flow.useNPMPackagedFlow": true,
   "javascript.validate.enable": false,
   "jest.pathToJest": "yarn jest --",
-  "prettier.eslintIntegration": true
+  "prettier.eslintIntegration": true,
+  "typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false
```

Comment:
roll back changes in this file (there were some fixes to autoformat yesterday (#9584), should fix the issue you had)